public/index.html

```diff
@@ -1,3 +1,187 @@
+                        </div>
+                        <div class="control">
+                            <button @click="pause" class="button is-warning">Pause</button>
+                        </div>
+                        <div class="control">
+                            <button @click="resume" class="button is-link">Resume</button>
+                        </div>
+                        <div class="control">
+                            <button @click="cancel" class="button is-danger">Stop</button>
+                        </div>
+                        <div class="control">
+                            <button @click="reset" class="button is-primary">Ask Another Question</button>
+                        </div>
+                    </div>
+                </div>
+            </section>
+            <div class="modal" :class="{ 'is-active': loading }">
+                <div class="modal-background"></div>
+                <div class="modal-content">
+                    <div class="container">
+                        <progress class="progress is-small is-primary" max="100">15%</progress>
+                    </div>
+                </div>
+            </div>
+        </div>
+        <script src='assets/js/axios.min.js'></script>
+        <script src='assets/js/vue.min.js'></script>
+        <script>
+        const app = new Vue({
+            el: "#app",
+            data: {
+              step: 1,
+              loading: false,
+              synth: null,
+              voices: null,
+              selectedVoiceName: 'Alex',
+              pitch: 1,
+              rate: 1,
+              question: '',
+              result: ''
+            },
+            mounted() {
+              this.synth = window.speechSynthesis;
+              this.getVoices().
+                then(voices => {
+                return this.voices = voices;
+                });
+            },
+            computed: {
+              selectedVoice() {
+                return this.voices.find(voice => voice.name === this.selectedVoiceName);
+              },
+              formInvalid() {
+                return this.question === '';
+              },
+            },
+            methods: {
+              getVoices() {
+                return new Promise((resolve, reject) => {
+                  let id;
+                  id = setInterval(() => {
+                    if (this.synth.getVoices().length !== 0) {
+                      resolve(this.synth.getVoices());
+                      clearInterval(id);
+                    }
+                  }, 10);
+                });
+              },
+              submit(response) {
+                this.loading = true;
+                axios.post('/architecture', {
+                  question: this.question
+                }).
+                  then(response => {
+                    this.step = 2;
+                    this.result = `You should use ${response.data.words.join(', ')}`;
```

Comment:
Can you add an "and" just before the last "word"?

Reply:
Totally. This was just the first iteration.